src/snyk/base/services/authenticationService.ts

```diff
@@ -106,6 +112,15 @@ export class AuthenticationService implements IAuthenticationService {
 
     if (!token) return;
     await this.configuration.setToken(token);
+    // TODO remove feature flag when ready
+    if (!this.configuration.getPreviewFeatures().lsAuthenticate) {
```

Comment:
fyi You can write it nicer by sending notification within an if. No need for a change though :)

Reply:
Wrote it this way to make it easier to find and to remove everywhere, once we remove the feature flag.

Reply:
You're one step ahead 🌟